src/actions/exchangeActions.js

```diff
@@ -242,10 +242,12 @@ export const searchOffersAction = (fromAssetCode: string, toAssetCode: string, f
           .map((offer: Offer) => dispatch({ type: ADD_OFFER, payload: offer })),
       );
       // we're requesting although it will start delivering when connection is established
-      const { error } = await exchangeService.requestOffers(fromAddress, toAddress, fromAmount);
+      const response = await exchangeService.requestOffers(fromAddress, toAddress, fromAmount);
+      const responseError = get(response, 'error');
 
-      if (error) {
-        const message = error.message || 'Unable to connect';
+      if (responseError) {
+        const responseErrorMessage = get(responseError, 'response.data.error.message');
```

Comment:
In 2 lines above you're checking the 'response.error' while here you're grabbing the message from 'response.data.error'

Reply:
Yes, this is correct because on call exception we return `{ error: axiosErrorResponse }` of which `axiosErrorResponse` is AxiosErrorResponse type from which `response.data` is axios built response and 'error.message' in 'data` is value returned from back-end.

Reply:
Oh, you're right! Didn't notice that you're taking it from the `responseError` object, sorry